src/main/java/com/webperside/courseerpbackend/services/language/LanguageService.java

```diff
@@ -11,4 +11,6 @@ public interface LanguageService {
     Language findById(Long id);
     void update (Language language);
 
+    Boolean isDefault(Long Id);
+    Boolean hasLocalization(Long id);
```

Comment:
you don't need these methods. 
just add a new method for getting default language

Reply:
Modification implemented.